myapp/getClients.js

```diff
@@ -0,0 +1,17 @@
+const request = require('request');
+const app = require('./app.js');
+
+const getClients = function () { 
+  app.get('/clients', function (req, res) {
+    request('https://api.createsend.com/api/v3.1/clients.json', function (err, response, data) {
+      if(err) {
+        res.send(err);
+        return(err);
+      }
+      res.json(data); 
+    })
+     .auth('fc96dc9ded698bbfbfb975409a864a42', 'ayusht11', false);
```

Comment:
messed up indentation, remove 2 whitespaces. This chaining is similar to the .then chaining you did in Promise.